projects/dotcms-field-elements/src/components/dot-form/dot-form.tsx

```diff
@@ -113,9 +132,33 @@ export class DotFormComponent {
 
     private handleSubmit(event: Event): void {
         event.preventDefault();
-        this.onSubmit.emit({
-            ...this.value
-        });
+
+        fetch(SUBMIT_FORM_API_URL, {
+            method: 'POST',
+            headers: {
+                'Content-Type': 'application/json'
+            },
+            body: JSON.stringify({
+                stName: this.variable,
+                ...this.value
+            })
+        })
+            .then(async (response: Response) => {
+                if (response.status !== 200) {
+                    const error: DotCMSFormSubmitError = {
+                        message: await response.text(),
+                        status: response.status
+                    };
+                    throw error;
+                }
+                return response.text();
+            })
+            .then((_text: string) => {
+                // Go to success page
```

Comment:
where or who sets the `success` page?

Reply:
<img width="730" alt="Screen Shot 2019-07-22 at 11 31 49 AM" src="https://user-images.githubusercontent.com/751424/61652028-56906c00-ac74-11e9-9e09-3350375862f5.png">